package.json

```diff
@@ -41,7 +41,7 @@
     "graceful-fs": "4.2.3",
     "http-proxy-agent": "^2.1.0",
     "https-proxy-agent": "^2.2.3",
-    "iconv-lite": "0.5.0",
+    "iconv-lite": "ashtuchkin/iconv-lite#4c3eff9dac5c225f38500a633a0cfdc4ad6e0eb8",
```

Comment:
I had to do it temporarily, so the types of `iconv.getDecoder/iconv.getEncoder` are there. However I believe we will change it before merging.

There are couple of routes we can take here:
1. Use `iconv-lite-umd` module, which will have correct type definitions.
2. Since the APIs are there, only types are not we can temporarily use `as ...` with our own type definitions.

Personally I prefer option 1.